tests/core/test_expectation_suite.py

```diff
@@ -436,6 +436,7 @@ def _test_expectation_can_be_saved_after_added(self, context, expectation):
         assert expectation.column != updated_column_name
         expectation.column = updated_column_name
         expectation.save()
+        assert suite.expectations[0].column == updated_column_name
```

Comment:
Want to check state of in-memory suite BEFORE the subsequent call to `get_expectation_suite` here